gateway/1.4/modules/ROOT/pages/_partials/task-config-logs-third-party.adoc

```diff
@@ -75,27 +114,123 @@ Flex Gateway configuration files, go to the last step.
 . Stop your Flex Gateway and any replicas, using Ctrl+C.
 . Create a folder in the directory with your Flex Gateway configuration files and name it `app`.
 . Restart your Flex Gateway with an additional volume for the new `app` directory:
```

Comment:
There's a colon at the end, but I'm not seeing an image or sublist. Should it be a period?

Reply:
Is line 117 a sublist?

Reply:
I built this and it doesn't like quite right. Can you retest?